client/reader/discover/components/add-new/style.scss

```diff
@@ -1,7 +1,17 @@
-@import "@automattic/typography/styles/variables";
+@import '@automattic/typography/styles/variables';
 
 .discover-add-new {
 	padding: 10px;
+
+	.notice-banner {
+		margin-bottom: 20px;
+	}
```

Comment:
Is this `notice-banner` class being used? 🤔
I realized I think it's unnecessary as I was implementing this for the Reddit tab.